### PR TITLE
Modify pickup collection behaviour to correspond to vanilla.

### PR DESCRIPTION
**Pull request by bearbin**
As documented here: https://www.spigotmc.org/threads/item-pickup-radius.337271/#post-3141146

### CODE REVIEW

**Comment by tigerw** [comment]
Thank you for fixing this, it was unbelievably unsatisfying when testing arrows that shooting them into the ceiling made them uncollectable.

File: src/Chunk.cpp [hunk 1]
```diff
@@ -1643,21 +1643,17 @@ void cChunk::SetAreaBiome(int a_MinRelX, int a_MaxRelX, int a_MinRelZ, int a_Max
 
 void cChunk::CollectPickupsByPlayer(cPlayer & a_Player)
 {
-	double PosX = a_Player.GetPosX();
-	double PosY = a_Player.GetPosY();
-	double PosZ = a_Player.GetPosZ();
+	auto BoundingBox = cBoundingBox(a_Player.GetPosition(), a_Player.GetWidth(), a_Player.GetHeight());
```
Comment (peterbell10) [inline src/Chunk.cpp:1646]: Side note: there should totally be a `cEntity::GetBoundingBox`.

Comment (bearbin) [reply src/Chunk.cpp:1646]: I thought of that, but there's only one other place in the whole codebase where it would be used. Still worth it?

Comment (peterbell10) [reply src/Chunk.cpp:1646]: I'd say yes. It also looks like it should actually be `GetWidth()/2.0`:
https://github.com/cuberite/cuberite/blob/0b9b7bc1a8d5cf6f92b802dc376a189ef066e62d/src/Entities/Minecart.cpp#L57

Comment (peterbell10) [reply src/Chunk.cpp:1646]: Unless that was intentionally twice the entity radius?

Comment (bearbin) [reply src/Chunk.cpp:1646]: No, it should just have been the entity radius. I guess with only 0.4 blocks difference it behaves basically the same anyway. I'll make a new PR to fix that and create a new method to get the bounding box.

**Pull request merged**